src/main/java/com/endava/webhw/dto/DepartmentDto.java

```diff
@@ -0,0 +1,22 @@
+package com.endava.webhw.dto;
+
+import com.endava.webhw.model.Department;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class DepartmentDto {
+    private String name;
+
+    private String location;
```

Comment:
Homework had something with validation of DTO

Reply:
Added validation to DTO according to the requirements.